android/src/main/kotlin/com/abilia/system_settings_editor/SystemSettingsHandler.kt

```diff
@@ -103,17 +103,22 @@ class SystemSettingsHandler(private val context: Context) {
   }
 
   internal fun setScreenOffTimeoutHandler(call: MethodCall, result: MethodChannel.Result) {
-    val timeout: Long? = call.argument("timeout")
+    val timeout: Any? = call.argument("timeout")
     timeout?.let {
-      setScreenOffTimeout(it)
+      val intVal = if (it is Long) {
+        Int.MAX_VALUE
```

Comment:
Why INT_MAX ? Is it because dart makes it long if it is to big?

If so a comment would've been nice